tests/unit/components/playback/controls/RecordingTime.spec.js

```diff
@@ -93,7 +93,7 @@ describe("RecordingTime.vue", () => {
       "00:00:55.018"
     );
   });
-  test("Given that Get method is mocked with a http response 200, When Recording start is initiated at x_time_index moved by 6736300, Then the text updates 00:00:55.018 and stop_recording initiates resets text to null", async () => {
+  test("Given that get method is mocked with a http response 200, When Recording start is initiated at x_time_index moved by 6736300, Then the text updates 00:00:55.018 and stop_recording initiates resets text to null", async () => {
```

Comment:
the `x_time_index` is not moved by `6736300`.  it is set to `6736300`.  This test is specifically testing that the Recording Time does not just convert the `x_time_index` itself into seconds to display (x_time_index is the total time since Live View started), but that the component only takes the difference in time from when the recording started and now to display.  6736300 centimilliseconds would be 1 minute 7.363 seconds, not 55.018 seconds

Reply:
Rectified as x_time_index is `mutated` to `6767300`.